content/docs/user-guide/assets/asset-processor/move-assets.md

```diff
@@ -59,4 +59,32 @@ You can use the additional command line options below when you move or delete as
 | `--leaveEmptyFolders` | By default, empty folders are removed, even during a `--move`. This option keeps empty folders. |
 | `--allowBrokenDependencies` |  By default, move and delete commands fail if there are broken dependencies. This forces the action to proceed despite broken dependencies. |
 | `--updateReferences` | This option is only available with the `--move` command. The option attempts to update files that reference the selected files. This is a simple find-and-replace of the absolute path and UUID/AssetId. References in binary assets are not updated. |
-| `--enablescm` | Source Control Management is disabled by default in Asset Processor Batch. This enables the source control plugin so that the command can checkout files for edit/move/delete as appropriate. |
\ No newline at end of file
+| `--enablescm` | Source Control Management is disabled by default in Asset Processor Batch. This enables the source control plugin so that the command can checkout files for edit/move/delete as appropriate. |
+
+## Asset Processor Batch Relocation Limitations
+<a name="Limitations"></a>
```

Comment:
Prefer to modify the header permalink rather than add HTML.
```suggestion
## Asset Processor Batch Relocation Limitations {#Limitations}
```

Reply:
I've updated the pull request with all these suggested changes